NEWS.rst

```diff
@@ -38,6 +41,7 @@ e.g. motifs.write(motifs, "clusterbuster", precision=2)
 Many thanks to the Biopython developers and community for making this release
 possible, especially the following contributors:
 
+- Samuel Prince (first contribution)
```

Comment:
This should be alphabetical by first name, although it looks like that needs fixing anyway.